model/derived/gasoutputs.go

```diff
@@ -0,0 +1,70 @@
+package derived
+
+import (
+	"context"
+
+	"github.com/go-pg/pg/v10"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/label"
+	"golang.org/x/xerrors"
+)
+
+type GasOutputs struct {
+	tableName     struct{} `pg:"derived_gas_outputs"`
+	Cid           string   `pg:",pk,notnull"`
+	From          string   `pg:",notnull"`
+	To            string   `pg:",notnull"`
+	Value         string   `pg:",notnull"`
+	GasFeeCap     string   `pg:",notnull"`
+	GasPremium    string   `pg:",notnull"`
+	GasLimit      int64    `pg:",use_zero"`
+	SizeBytes     int      `pg:",use_zero"`
+	Nonce         uint64   `pg:",use_zero"`
+	Method        uint64   `pg:",use_zero"`
+	Params        []byte
+	StateRoot     string `pg:",notnull"`
+	Idx           int    `pg:",use_zero"`
+	ExitCode      int64  `pg:",use_zero"`
+	GasUsed       int64  `pg:",use_zero"`
+	Return        []byte
+	ParentBaseFee string
+
+	BaseFeeBurn        string
+	OverEstimationBurn string
+
+	MinerPenalty string
+	MinerTip     string
+	Refund       string
+
+	GasRefund int64
+	GasBurned int64
+}
```

Comment:
I think all of these columns should be `notnull` (even the cases which `use_zero`).

Reply:
Done